some users can't compile Machida in OS X High Sierra (10.13)

## Description

Some OS X users (such as me) can't compile Machida in OS High Sierra. When I try to compile it with `make` I get the following error:

```
/Users/aturley/development/wallaroo.word_count_demo/machida/cpp/python-wallaroo.c:21:18: fatal error: 'Python/Python.h' file not found
        #include <Python/Python.h>
                 ^~~~~~~~~~~~~~~~~
1 error generated.
make: *** [/Users/aturley/development/wallaroo.word_count_demo/machida/build/python-wallaroo.o] Error 1
```

## Steps to Reproduce

1. Clone Wallaroo on an OS X High Sierra machine.
2. Go to the Machida directory
3. Run `make`

## Expected Behavior

Machida builds.

## Actual Behavior

Machida does not build, and the compiler reports the error above.

--- a/machida/cpp/python-wallaroo.c
+++ b/machida/cpp/python-wallaroo.c
@@ -17,6 +17,7 @@ Copyright 2017 The Wallaroo Authors.
 */
 
 #ifdef __APPLE__
+    #include <AvailabilityMacros.h>
     #if MAC_OS_X_VERSION_MAX_ALLOWED < 101300
         #include <Python/Python.h>
     #else

